addons/skin.estuary/xml/DialogVideoInfo.xml

```diff
@@ -411,89 +433,91 @@
 							</control>
 						</control>
 					</focusedlayout>
-					<itemlayout height="317" width="245" condition="Container.Content(Sets)">
+					<itemlayout height="317" width="210" condition="Container.Content(Sets)">
 						<control type="group">
+							<left>-12</left>
```

Comment:
hmm... why a different offset compared to the focusedlayout?

Reply:
Not sure why the offset is needed, but if I don't use that offset I get the following:

![image](https://user-images.githubusercontent.com/7235787/86875132-1bd04f80-c0e2-11ea-9328-b4f8353e00b0.png)

Reply:
looks like you've moved most controls in the itemlayout 10px to the right.. and then compensate that with the -12px offset.
mind having another look at that?

the positioning of the controls in the item & focused layout should be identical.